edenai_apis/apis/together_ai/together_ai_api.py

```
@@ -8,9 +11,11 @@
 from edenai_apis.utils.types import ResponseType
 from edenai_apis.loaders.loaders import load_provider
 from edenai_apis.loaders.data_loader import ProviderDataEnum
+from edenai_apis.features.llm.llm_interface import LlmInterface
+from edenai_apis.features.llm.chat.chat_dataclass import ChatDataClass
```

Ruff (0.8.2)

Redefinition of unused `ChatDataClass` from line 7

(F811)